controllers/admin/AdminSeur15Controller.php

```diff
@@ -357,28 +375,30 @@ public function printLabels($id_orders = array(), $type = 'txt')
 							echo $fp;				
 						}
 					}
-		}
-		elseif ($type == 'pdf')
-		{
-			if (file_exists($directory.$name.'.pdf') && ($fp = Tools::file_get_contents($directory.$name.'.pdf')))
+			}
+			elseif ($type == 'pdf')
 			{
-				ob_end_clean();
-				header('Content-type: application/pdf');
-				header('Content-Disposition: inline; filename='.$name.'.pdf');
-				header('Content-Transfer-Encoding: binary');
-				header('Accept-Ranges: bytes');
+				if (file_exists($directory.$name.'.pdf') && ($fp = Tools::file_get_contents($directory.$name.'.pdf')))
+				{
+					ob_end_clean();
+					header('Content-type: application/pdf');
+					header('Content-Disposition: inline; filename='.$name.'.pdf');
+					header('Content-Transfer-Encoding: binary');
+					header('Accept-Ranges: bytes');
 
-				echo $fp;
+					echo $fp;
+				}
 			}
-		}
 		
-		}catch(Exception $e)
+		}
+		catch(Exception $e)
 		{
 			echo $e->getMessage();
 		}
 		exit;	
 		$this->context->smarty->assign('error', $this->l('Document was already printed, but is missing in module directory'));
```

Comment:
You have to specify the class name in the second parameter of translation:
$this->l('my sentence', 'myClassName');